xbmc/pvr/epg/EpgDatabase.cpp

```diff
@@ -371,9 +466,9 @@ int CPVREpgDatabase::Persist(const CPVREpgInfoTag& tag, bool bSingleUpdate /* =
   tag.StartAsUTC().GetAsTime(iStartTime);
   tag.EndAsUTC().GetAsTime(iEndTime);
 
-  time_t iFirstAired = 0;
-  if (tag.FirstAiredAsUTC().IsValid())
-    tag.FirstAiredAsUTC().GetAsTime(iFirstAired);
+  std::string sFirstAired;
```

Comment:
const std::string

Reply:
This one can't be const as it's subsequently assigned a value if .FirstAired().IsValid().  I can make it inline, but I think it would look uglier?

```
  const std::string sFirstAired(
      (tag.FirstAired().IsValid() ? tag.FirstAired().GetAsW3CDate() : ""));
```

Reply:
No, was just an oversight on my end. Sorry.